sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -777,6 +778,29 @@ public sealed partial class TrueRuleFilter : Azure.Messaging.ServiceBus.Manageme
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Messaging.ServiceBus.Plugins
+{
+    public abstract partial class ServiceBusPlugin
+    {
+        protected ServiceBusPlugin() { }
+        public virtual System.Threading.Tasks.Task AfterMessageReceive(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message) { throw null; }
+        public virtual System.Threading.Tasks.Task BeforeMessageSend(Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
```

Comment:
Should these be async?

Reply:
Yeah, they should be suffixed with Async. Good catch.